mutiny-core/src/lib.rs

```diff
@@ -119,6 +119,7 @@ use crate::utils::parse_profile_metadata;
 use mockall::{automock, predicate::*};
 
 const DEFAULT_PAYMENT_TIMEOUT: u64 = 30;
+const DEFAULT_REISSUE_TIMEOUT: u64 = 50;
```

Comment:
this is only used in federation.rs, would be better to define in there. This should probably go back to 30 seconds too. you're also always casting to `i32`, can just define it as such

Reply:
nice, done!
but I kept it as u64 to follow the standard, or should I also change the other ones? 🤔